components/authentication/base/group-sync/external-secrets/kustomization.yaml

```diff
@@ -1,5 +1,4 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-  - github-redhat-appstudio-sre.yaml
```

Comment:
group 'stage' is synced from this -> components/authentication/base/everyone-can-view.yaml so it should be removed or replaced.

Reply:
thanks, I'll remove it. I think it's to wide.

Reply:
ok, lets keep the redhat-appstudio-sre sync for now since it's useful for debugging.